src/main/java/org/sagebionetworks/bridge/spring/controllers/SponsorController.java

```diff
@@ -53,6 +54,18 @@
         
         return sponsorService.getSponsoredStudies(session.getAppId(), orgId, offsetByInt, pageSizeInt);
     }
+
+    @GetMapping("/v1/organizations/{orgId}/apps/{appId}/studies")
+    public PagedResourceList<Study> getSponsoredStudiesForWorker(@PathVariable String appId, @PathVariable String orgId,
```

Comment:
Consider renaming this to getSponsoredStudiesForApp(), which better describes what it is used for.